packages/terminal/src/browser/terminal-quick-open-service.ts

```diff
@@ -0,0 +1,174 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import {
+    QuickOpenModel, QuickOpenGroupItem, QuickOpenHandler,
+    QuickOpenOptions, QuickOpenItemOptions, QuickOpenMode,
+    PrefixQuickOpenService,
+    QuickOpenContribution, QuickOpenHandlerRegistry, QuickOpenGroupItemOptions
+} from '@theia/core/lib/browser';
+import { CommandContribution } from '@theia/core/lib/common';
+import { CommandRegistry } from '@theia/core/lib/common';
```

Comment:
CommandContribution and CommandRegistry are from the same lib, could be put in a single line/
import  {...} ...

Reply:
@fangnx could you provide this minor fix and I'll merge the PR :)